### PR TITLE
docs: Daft integration

### LINKED ISSUE #127: Create Daft documentation integration page
The Daft team gave an awesome presentation on their integration to the UC Community: https://www.youtube.com/live/8i5eOE3KdBo?feature=shared&t=458

Would be great to create a Daft integration page in the docs.

### PR DESCRIPTION
**Pull request by avriiil**
This adds a documentation page for Daft, will close #127

### CODE REVIEW

**Review by MrPowers — APPROVED** [review]
LGTM.  This is amazing.  Wonderful work Avril!

File: docs/integrations/unity-catalog-daft.md [hunk 1]
```diff
@@ -1,3 +1,119 @@
 # Unity Catalog Daft Integration
 
-TODO
+This page shows you how to use Unity Catalog with Daft.
```
Comment (tdas) [inline docs/integrations/unity-catalog-daft.md:3]: I think it makes more sense to name this file just `daft.md`. everything is in unitycatalog and so it does make sense to have the prefix `unity-catalog-` in an individual page with the url being `docs.unitycatalog.io/integrations/unity-catalog-daft/`. Just `docs.unitycatalog.io/integrations/daft/` makes more sense.

**Review by tdas — CHANGES_REQUESTED** [review]
please change the file name as i suggested. thank you for your contribution!

**Review by tdas — APPROVED** [review]
approving this PR to unblock @MrPowers from making more changes. lets make sure my comment is addressed before going live

**Pull request merged**